src/ui/components/Initiatives/InitiativeCreate.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import React from "react";
+import {v4 as uuid} from "uuid";
+import {
+  AutocompleteArrayInput,
+  DateInput,
+  ArrayInput,
+  NumberInput,
+  SimpleForm,
+  TextInput,
+  Create,
+  SimpleFormIterator,
+  BooleanInput,
+} from "react-admin";
+import {
+  getPlainDescFromMd,
+  dateFormatter,
+  dateParser,
+} from "../../initiativeUtils";
+import {CredView} from "../../../analysis/credView";
+
+export const InitiativeCreate = (credView: CredView) => (props: Object) => {
+  const defaultInitiativeValues = {
+    id: uuid(),
+    timestampMs: Date.now(),
+    champions: [],
+    dependencies: [],
+    references: [],
+    contributions: [],
+    weight: {incomplete: 0, complete: 0},
+    completed: false,
+  };
+  return (
+    <Create title="Create New Initiative" {...props}>
+      <SimpleForm initialValues={defaultInitiativeValues}>
+        <TextInput
+          source="id"
+          label="Initiative ID"
+          disabled
+          type="hidden"
+          style={{display: "none"}}
```

Comment:
Do we really want to hide it? Having it visible will allow us to debug and make sure its working correctly, can still have it disabled. If we do want to hide it, should just setup a css class that we can specify instead of having inline style objects

Reply:
addressed this in #1968